README.md

```diff
@@ -265,6 +271,60 @@ class MyStack : Stack
 }
 ```
 
+### Java
+
+```console
+kube2pulumi java -f ./pod.yaml
+```
+
+```java
+package generated_program;
+
+import com.pulumi.Context;
+import com.pulumi.Pulumi;
+import com.pulumi.core.Output;
+import com.pulumi.kubernetes.core_v1.Pod;
+import com.pulumi.kubernetes.core_v1.PodArgs;
+import com.pulumi.kubernetes.meta_v1.inputs.ObjectMetaArgs;
+import com.pulumi.kubernetes.core_v1.inputs.PodSpecArgs;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
```

Comment:
```suggestion
import com.pulumi.Context;
import com.pulumi.Pulumi;
import com.pulumi.kubernetes.core_v1.Pod;
import com.pulumi.kubernetes.core_v1.PodArgs;
import com.pulumi.kubernetes.meta_v1.inputs.ObjectMetaArgs;
import com.pulumi.kubernetes.core_v1.inputs.PodSpecArgs;
import java.util.Map;
```

These are the only required imports.

NOTE: I didn't add the missing `ContainerArgs` import, just trying to showcase the added unused imports.